roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -375,6 +375,7 @@ openstack_blockstorage_ignore_volume_az: "{{ volume_cross_zone_attachment | defa
 # Set Cinder topology zones (can be multiple zones, default not set)
 # cinder_topology_zones:
 #   - nova
+cinder_ignore_volume_az: "{{ volume_cross_zone_attachment | default('false') }}"
```

Comment:
I guess the same option has been already implemented as openstack_blockstorage_ignore_volume_az at line 370.
And the value is used in roles/kubernetes/node/templates/cloud-configs/openstack-cloud-config.j2 for ignore-volume-az option.
Is that different one?

Reply:
Thanks for your comment @oomichi .I did not specify it in the PR description, this is for cinder-csi, while openstack_blockstorage_ignore_volume_az is for traditional cloud provider OpenStack.

Reply:
Why don't we pass the existing openstack_blockstorage_ignore_volume_az option value for the cinder-csi also?
My point is it is easy to make confusion between openstack_blockstorage_ignore_volume_az and cinder_ignore_volume_az.
It would be necessary to write some comment for this new option at least to explain what is different from the other one.

Reply:
Good question @oomichi . I agree it's confusing. I added some documentation.

The variables `cinder_blockstorage_version` exists alongside `openstack_blockstorage_version`.
Same for other OpenStack variables, they exist under the form `cinder_` and `openstack_`. 
Therefore I followed the same  approach for this one.